agg-ast/schema_derivation/src/schema_derivation_tests/stage.rs

```diff
@@ -164,6 +164,116 @@ mod add_fields {
     );
 }
 
+mod bucket {
+    use super::*;
+
+    test_derive_stage_schema!(
+        bucket_no_output,
+        expected = Ok(Schema::Document(Document {
+            keys: map! {
+                "_id".to_string() => Schema::Atomic(Atomic::String),
+                "count".to_string() => Schema::AnyOf(set!(
+                    Schema::Atomic(Atomic::Integer),
+                    Schema::Atomic(Atomic::Long)
+                ))
+            },
+            required: set!("_id".to_string(), "count".to_string()),
+            ..Default::default()
+        })),
+        input = r#"{"$bucket": {"groupBy": "$foo", "boundaries": ["hello", "world", "zod"], "buckets": 5}}"#,
```

Comment:
In the documentation I see that the `buckets` parameter is specified for the `$bucketAuto` stage and not `$bucket`.  Should we remove it here and in the `bucket_no_output_and_default` test for correctness?

Reply:
Oh yes, that was a mistake. I'm surprised it parsed.